README.md

```diff
@@ -99,25 +61,25 @@ easier-to-extend interface. If you'd like to browse the old toolchain, its
 status is recorded for posterity at
 https://github.com/kivy/python-for-android/tree/old_toolchain.
 
-In the last quarter of 2018 the python recipes were changed. The
-new recipe for python3 (3.7.1) had a new build system which was
-applied to the ancient python recipe, allowing us to bump the python2
+In the last quarter of 2018 the Python recipes were changed. The
+new recipe for Python3 (3.7.1) had a new build system which was
+applied to the ancient Python recipe, allowing us to bump the Python2
 version number to 2.7.15. This change unified the build process for
-both python recipes, and probably solved various issues detected over the
-years. These **unified python recipes** require a **minimum target api level of 21**,
+both Python recipes, and probably solved various issues detected over the
+years. These **unified Python recipes** require a **minimum target api level of 21**,
 *Android 5.0 - Lollipop*. If you need to build targeting an
```

Comment:
I'm wondering if we want to keep the history section.
On one hand it may help core contributors to understand some technical choices, on the other hand I'm not sure the README.md is the right place

Reply:
I understand the dilemma, but there is an easy solution. The Contributing Guidelines are already on the operating table! I have moved the history section into the Contributing Guidelines where it is right there for the people about to contribute.

(If we ever created an "Technical Architecture" section, it might do better there, but this is the best place for now.)